migrate(docs): troubleshooting 1 (#30866)
supabase/supabase: apps/docs/content/troubleshooting/why-is-my-camelcase-name-not-working-in-postgres-functions-or-rls-policies-EJMzVd.mdx

---
title = "Why is my camelCase name not working in Postgres functions or RLS policies?"
github_url = "https://github.com/orgs/supabase/discussions/12893"
date_created = "2023-03-08T16:26:51+00:00"
topics = ["database"]
---

> Avoid camelCase (upper case letters) if it all possible in Postgres for naming of functions, tables, columns, vars. You must enclose "camelCase" in double quotes when you define it or use it. YOU WILL FORGET. Use snake_case instead and make your life easier.

Reviewer: at?
```suggestion
Avoid camelCase (upper case letters) at all if possible in Postgres for naming functions, tables, columns, vars. You must enclose "camelCase" in double quotes when you define it or use it. YOU WILL FORGET. Use snake_case instead and make your life easier.
```

Reviewer: Or even removing "at all" (if _at all_ was what you wanted to write)

Author: ```suggestion
Avoid camelCase (upper case letters) if possible when naming functions, tables, columns, and variables. You must enclose "camelCase" in double quotes when you define it or use it. YOU WILL FORGET. Use snake_case instead and make your life easier.
```
